lib/asciidoctor/rx.rb

```diff
@@ -0,0 +1,728 @@
+  # NOTE { is included in start characters to support target that begins with attribute reference in title content
+  InlineXrefMacroRx = %r(\\?(?:&lt;&lt;([#{CC_WORD}#/.:{]#{CC_ALL}*?)&gt;&gt;|xref:([#{CC_WORD}#/.:{]#{CC_ALL}*?)\[(?:\]|(#{CC_ALL}*?[^\\])\])))m
+
+  ## Layout
+
+  # Matches a trailing + preceded by at least one space character,
+  # which forces a hard line break (<br> tag in HTML output).
+  #
+  # NOTE AsciiDoc Python allows + to be preceded by TAB; Asciidoctor does not
+  #
+  # Examples
+  #
+  #   Humpty Dumpty sat on a wall, +
+  #   Humpty Dumpty had a great fall.
+  #
+  if RUBY_ENGINE == 'opal'
+    # NOTE In JavaScript, ^ and $ only match the start and end of line if the multiline flag is present
+    HardLineBreakRx = /^(#{CC_ANY}*) \+$/m
+  else
+    # NOTE In Ruby, ^ and $ always match start and end of line
+    HardLineBreakRx = /^(.*) \+$/
+  end
+
+  # Matches a Markdown horizontal rule.
+  #
+  # Examples
+  #
+  #   --- or - - -
+  #   *** or * * *
+  #   ___ or _ _ _
+  #
+  MarkdownThematicBreakRx = /^ {0,3}([-*_])( *)\1\2\1$/
+
+  # Matches an AsciiDoc or Markdown horizontal rule or AsciiDoc page break.
+  #
+  # Examples
+  #
+  #   ''' (horizontal rule)
+  #   <<< (page break)
+  #   --- or - - - (horizontal rule, Markdown)
+  #   *** or * * * (horizontal rule, Markdown)
+  #   ___ or _ _ _ (horizontal rule, Markdown)
+  #
+  ExtLayoutBreakRx = /^(?:'{3,}|<{3,}|([-*_])( *)\1\2\1)$/
+
+  ## General
+
+  # Matches consecutive blank lines.
+  #
+  # Examples
+  #
+  #   one
+  #
+  #   two
+  #
+  BlankLineRx = /\n{2,}/
+
+  # Matches a comma or semi-colon delimiter.
+  #
+  # Examples
+  #
+  #   one,two
+  #   three;four
+  #
+  #DataDelimiterRx = /[,;]/
+
+  # Matches whitespace (space, tab, newline) escaped by a backslash.
+  #
+  # Examples
+  #
+  #   three\ blind\ mice
+  #
+  EscapedSpaceRx = /\\([ \t\n])/
+
+  # Detects if text is a possible candidate for the replacements substitution.
+  #
+  ReplaceableTextRx = /[&']|--|\.\.\.|\([CRT]M?\)/
+
+  # Matches a whitespace delimiter, a sequence of spaces, tabs, and/or newlines.
+  # Matches the parsing rules of %w strings in Ruby.
+  #
+  # Examples
+  #
+  #   one two	 three   four
+  #   five	six
+  #
+  # TODO change to /(?<!\\)[ \t\n]+/ once lookbehind assertions are implemented in all modern browsers
+  SpaceDelimiterRx = /([^\\])[ \t\n]+/
+
+  # Matches a + or - modifier in a subs list
+  #
+  SubModifierSniffRx = /[+-]/
+
+  # Matches one or more consecutive digits at the end of a line.
+  #
+  # Examples
+  #
+  #   docbook5
+  #   html5
+  #
+  TrailingDigitsRx = /\d+$/
+
+  # Detects strings that resemble URIs.
+  #
+  # Examples
+  #   http://domain
+  #   https://domain
+  #   file:///path
+  #   data:info
+  #
+  #   not c:/sample.adoc or c:\sample.adoc
+  #
+  UriSniffRx = %r(^#{CG_ALPHA}[#{CC_ALNUM}.+-]+:/{0,2})
+
+  # Detects XML tags
+  XmlSanitizeRx = /<[^>]+>/
+#end
```

Comment:
?

Reply:
It's the end of the pseudo module Rx, though at this point I think it's clear we decided not to use a submodule and thus we can remove the whole pseudo block.